src/__experimental__/components/search/search.style.js

```diff
@@ -59,7 +96,27 @@ const StyledSearch = styled.div`
         padding-bottom: 2px;
         padding-top: 1px; 
         cursor: pointer;
-      }
+          ${
+            !isFocused &&
+            !searchHasValue &&
+            css`
+              border: 1px solid transparent;
+              color: ${theme.icon.default};
+            `
+          }
+          ${
+            !isFocused &&
+            searchHasValue &&
+            !searchButton &&
```

Comment:
this set of conditionals is repeated three times in this file, what do you think of making it a const?

Reply:
That sounds fine to me. I had thought about making it a `const` but decided against it as the other conditionals were set out this way. I'll get this addressed 👍